go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/caelifer/gorebuild
+
+go 1.14
```

Comment:
I'd recommend go 1.11 unless an higher version is needed (?). It's useful to be able to rebuild while changing the current Go toolchain version to test specific projects on older ones. That'd be nice to actually test that this works (via travis) but that's out of scope for this CL.

Since you mainly want to fix os.Rename(), I'd recommend to leave this file out.

Reply:
The module support was added unintentionally and was removed. Thanks for spotting this up.